src/mod/applications/mod_conference/conference_event.c

```diff
@@ -108,7 +108,8 @@ void conference_event_mod_channel_handler(const char *event_channel, cJSON *json
 		}
 	}
 
-	switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_DEBUG, "conf %s CMD %s [%s] %s\n", conference_name, key, action, cid);
+//	switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_DEBUG, "conf %s CMD %s [%s] %s\n", conference_name, key, action, cid);
```

Comment:
please remove the commented out line from patch.

Reply:
52d94ae removes the commented out line and restore all whitespaces. 

Thanks.